db/migrate/20190813030005_add_column_to_datatables.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToDatatables < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :datatables, :key, :series
```

Comment:
Slightly worried about this renaming, as I need to check if we have used "key" anywhere in our current codebase. Stuff might break and be difficult to find if i immediately merge this